File: Samples/Android-Advanced/app/src/main/java/com/pushwoosh/sample/SummaryNotificationFactorySample.java
package com.pushwoosh.sample;

import com.pushwoosh.notification.PushwooshSummaryNotificationFactory;

public class SummaryNotificationFactorySample extends PushwooshSummaryNotificationFactory {

    @Override
    public String summaryNotificationMessage(int messagesCount) {
        return messagesCount + " new messages";
    }

    @Override
    public int summaryNotificationIconResId() {
        return R.drawable.ic_notification;
    }
}

File: Samples/Android-Advanced-Kotlin/app/src/main/java/com/pushwoosh/sample/SummaryNotificationFactorySample.kt
package com.pushwoosh.sample

import com.pushwoosh.notification.PushwooshSummaryNotificationFactory

class SummaryNotificationFactorySample : PushwooshSummaryNotificationFactory() {
    override fun summaryNotificationMessage(messagesCount: Int): String {
        return "$messagesCount new messages"
    }

    override fun summaryNotificationIconResId(): Int {
        return R.drawable.ic_notification
    }
}